package.json

```diff
@@ -137,8 +137,8 @@
     "build:lib": "babel --version && node ./bin/build-lib.js",
     "build": "npm-run-all --parallel build:js build:css --serial build:gzip size",
     "clean": "rm -rf lib && rm -rf dist",
-    "lint:fix": "eslint src test website/scripts website/build-examples.js website/update.js website/themes/uppy/source/js/common.js --fix",
-    "lint": "eslint src test website/scripts website/build-examples.js website/update.js website/themes/uppy/source/js/common.js",
+    "lint:fix": "npm run lint -- --fix",
+    "lint": "eslint packages/*/src packages/@uppy/*/src test website/scripts website/build-examples.js website/update.js website/themes/uppy/source/js/common.js",
```

Comment:
As opposed to whitelisting dirs here, may i propose doing `eslint .` and adding a blacklist in `.eslintignore`? While setting it up, if it takes very long, `env DEBUG=eslint:cli-engine eslint .` will show you what files it is crunching on that you'll likely also want to blacklist.

Two reasons for doing it:
- those with eslint-enabled editors won't get warnings about files that we don't care about (.eslintignore is parsed while this npm script is not)
- when we add new things, we'll have to opt-out of linting, vs opting-in, which feels safer

Reply:
yeah that sounds bettere :+1:

Reply:
And upon a closer look i think  `npm run ` can be removed

Reply:
oh maybe not actually :) nvm